examples/query-subscription-consumer/1/query-subscription-consumer

```diff
@@ -0,0 +1,19 @@
+{
+  "version": 3,
+  "payload": {
+    "subscription_id": "1234", 
```

Comment:
Is this the actual format of a subscription ID? Isn't it supposed to look like a partition number and UUID